packages/webpack-example-lite/README.md

```diff
@@ -0,0 +1,29 @@
+## 0x Mesh Browser Example
```

Comment:
This page is full of out-of-date links. We should also add a line near the top explaining the difference between this example and __../webpack-example__.